playbooks/roles/streisand-gateway/tasks/main.yml

```diff
@@ -128,6 +128,12 @@
         state=directory
         recurse=yes
 
+- name: Verify libreswan compilation has completed
```

Comment:
Libreswan does not get compiled and installed on AWS and GCE instances, so this task would fail on those providers. I'm also not sure how I feel about a Libreswan-related task being placed outside of the l2tp-ipsec role.